Dockerfile

```diff
@@ -24,15 +24,21 @@ RUN apt-get clean && apt-get update && apt-get install -y --no-install-recommend
     python3-dev \
     jq vim \
     ca-certificates \
+    gcc \
```

Comment:
It's an runtime dockerfile rather than for build. Here's the build dockerfile that including all the build tools.  https://github.com/TEN-framework/ten_agent_build